### DISCUSSION TITLE
Use this method to enable DSDS toggle without root/factory reset possible?

### DISCUSSION THREAD

**Discussion by alvgalrus** [kyujin-cho/pixel-volte-patch#184] [Ideas]
Hello!

In some phones using Android 13, there is an ongoing issue in the `*#*#4636#*#*` menu in which, when disabling the DSDS toggle, it disappears and cannot be enabled until after a factory reset is performed or a property is set with rooted adb.

Usually the adb way of doing this is documented as:
```
su
setprop persist.vendor.radio.multisim_switch_support true
setprop persist.radio.multisim.config dsds
```

I was wondering if this method could be used to bring back the DSDS toggle without either root or factory reset.

Thank you!

**Comment by mickyvirus007** [comment]
There are orther ways to fix this without pc. It only requires you to turn oem unlocking on in the developer options. You can check it out here https://onesectech.blogspot.com/2024/08/how-to-re-enable-dual-sim-dual.html
